README.md

```diff
@@ -375,26 +341,25 @@ To use Regula with Conftest:
 
 ### Directory structure
 
- -  `bin/`: the main Regula script that calls `terraform` & `opa`.
+ -  `bin/`: the main Regula script that, depending on the IaC platform, will call `opa` and `terraform` or convert CloudFormation YAML to JSON.
  -  `lib/`: the OPA library code to evaluate rules and mangle input.
- -  `rules/`: a collection of rules.  We may split this up further as the number
-    of rules increases.
+ -  `rules/`: a collection of rules.
  -  `examples/`: a collection of example rules that you can use as inspiration
      for your own rules.
  -  `scripts/`: scripts for development; currently only a script to generate
     test input.
  -  `tests/`:
       *  `tests/lib`: internal tests for the library.
       *  `tests/rules/`: tests for the various rules.
-      *  `tests/rules/inputs`: terraform files that can be used to generate Rego
+      *  `tests/rules/inputs`: CloudFormation and terraform files that can be used to generate Rego
```

Comment:
With the directory changes, looks like this will be the structure:

* `tests/rules/tf/<provider>/<service>/inputs` (Terraform files)
* `tests/rules/cfn/<service>/inputs` (CloudFormation files)